core/src/main/java/hudson/util/io/RewindableFileOutputStream.java

```diff
@@ -51,8 +54,8 @@ private synchronized OutputStream current() throws IOException {
             if (!closed) {
                 FileUtils.forceMkdir(out.getParentFile());
                 try {
-                    current = new FileOutputStream(out,false);
-                } catch (FileNotFoundException e) {
+                    current = Files.newOutputStream(out.toPath(), StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING);
+                } catch (FileNotFoundException | NoSuchFileException e) {
```

Comment:
Hmm, why? I would understand the InvalidPathException: https://docs.oracle.com/javase/7/docs/api/java/io/File.html#toPath()